public/js/actions/ArticleActions.js

```diff
@@ -0,0 +1,44 @@
+import dispatcher from '../dispatcher.js'
+
+export default class ArticleActions {
+    // TODO: Have different fetch calls for top/hot articles
+    static fetchArticles() {
+        return new Promise((resolve, reject) => {
+            fetch(`articles`)
+                .then(res => res.json())
+                .then((articles) => {
+                    articles = articles.sort((a, b) => b.votes - a.votes)
```

Comment:
How does the sorting work? Isn't it being passed a JSON string? It looks like its comparing the current article to the previous article, and using the difference in their votes.